ASPnetCoreMVC/Controllers/OrderController.cs

```diff
@@ -20,9 +20,9 @@ public OrderController(IProductService productService, ICategoryService category
             _categoryService = categoryService;
         }
 
-        public IActionResult LoadProductTable(string name, string priceFilter, string sortColumn, string sortOrder, int? categoryId)
+        public IActionResult LoadProductTable( int? categoryId)
         {
-            var products = _productService.GetAllProducts(name, priceFilter, sortColumn, sortOrder, categoryId);
+            var products = _productService.GetAllProducts("", "", "", "", categoryId);
```

Comment:
move categoryId as a first parameter, so the rest parameter can be optional paramers, we don't have to set it as ""